src/main/java/br/com/sunty/jdbc/dao/CourseDAO.java

```diff
@@ -0,0 +1,90 @@
+package br.com.sunty.jdbc.dao;
+
+import br.com.sunty.dto.CourseDTO;
+import br.com.sunty.models.course.Course;
+import br.com.sunty.models.course.CourseVisibility;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CourseDAO {
+
+    private Connection connection;
+
+    public CourseDAO(Connection connection) {
+        this.connection = connection;
+    }
+
+    public void saveCourse(Course course) throws SQLException {
```

Comment:
Se declararmos `SQLException`, estamos deixando vazar um detalhe de implementação da DAO para o mundo externo. Seria interessante embrulhar essa exception em uma DaoException, conforme mencionando anteriormente.

```suggestion
    public void saveCourse(Course course) {
```